README.md

```diff
@@ -155,20 +155,26 @@ publicKey=example_key
 
 * **publicKey** - this is the public key used to verify the cookie
 
-### Generating Keys
+### Rotating Keys
 
 **Guardian Devs**: See the [Emergency Panda key-rotation Guide](https://docs.google.com/document/d/1haVnQ9D8zNYUU-fOfkudPC1WpPGrlelLygd8V7xb3eQ/edit?usp=sharing).
```

Comment:
would it make sense to split out emergency rotation vs. routine rotation under separate headings?

Reply:
Actually, I think the word 'emergency' can be removed from a few parts of the documentation now, as, with this PR, key-rotation is no longer an emergency-only procedure! So I've added 56b456ecd952f5400499f3bbf14575c269b08375 to take out the word 'emergency' here, and once this PR is merged I'll be updating [_Emergency Panda key rotation instructions_](https://docs.google.com/document/d/1haVnQ9D8zNYUU-fOfkudPC1WpPGrlelLygd8V7xb3eQ/edit) to be titled just _Guardian Panda key rotation instructions_, and of course updating the instructions within too.